README.md

```diff
@@ -306,9 +311,68 @@ class ReportingOperation {
 register('send-report', ReportingOperation);
 ```
 
+### Retaining a worklet for multiple operations
+
+We may wish to run multiple worklet operations from the same context, e.g. we might select a URL and then send one or more aggregatable reports. To do so, we would need to use the `retain: true` option when calling eaching of our worklet operations (except perhaps in the last call, if we have no need to extend the worklet's lifetime beyond that call).
```

Comment:
editing nit, we seems a bit strange to me. Perhaps, "callers may wish to"? And then below would be something like, s/we could make the calls as follows/As an example:/

Reply:
Done.